presto-main-base/src/main/java/com/facebook/presto/sql/analyzer/StatementAnalyzer.java

```diff
@@ -2592,7 +2592,15 @@ private MaterializedViewStatus getMaterializedViewStatus(QualifiedObjectName mat
                     Map<String, Domain> rewrittenDomain = new HashMap<>();
 
                     for (Map.Entry<String, Domain> entry : viewQueryDomain.getDomains().orElse(ImmutableMap.of()).entrySet()) {
-                        Map<SchemaTableName, String> baseTableMapping = directColumnMappings.get(entry.getKey());
+                        String actualColumnName = null;
+                        for (String columnName : directColumnMappings.keySet()) {
+                            if (columnName.equalsIgnoreCase(entry.getKey())) {
+                                actualColumnName = columnName;
+                                break;
+                            }
+                        }
+
+                        Map<SchemaTableName, String> baseTableMapping = actualColumnName != null ? directColumnMappings.get(actualColumnName) : null;
```

Comment:
```suggestion
                        Map<SchemaTableName, String> baseTableMapping = null;
                        for (String columnName : directColumnMappings.keySet()) {
                            if (columnName.equalsIgnoreCase(entry.getKey())) {
                                baseTableMapping = directColumnMappings.get(columnName);
                                break;
                            }
                        }
```

nit: what do you think about avoiding declaring a variable `actualColumnName` here?